opentracing-tracerresolver/src/main/java/io/opentracing/contrib/tracerresolver/TracerResolver.java

```diff
@@ -85,13 +117,16 @@ public static Tracer resolveTracer() {
 
         Tracer tracer = null;
         if (!TracerResolver.isDisabled()) {
-            tracer = getFromFactory();
+            if (classloader == null) {
+        		classloader = Thread.currentThread().getContextClassLoader();
```

Comment:
Nit: indentation looks 'off'; could it be spaces vs tabs or did you add so many spaces intentionally?

Reply:
Good catch!  I used a new Eclipse workspace and forgot to change the default tab key to space.